src/main/java/org/tikv/common/log/SlowLogTaskImpl.java

```diff
@@ -0,0 +1,79 @@
+/*
+ *
+ * Copyright 2021 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.tikv.common.log;
+
+import static org.tikv.common.log.SlowLogImpl.DATE_FORMAT;
+
+public class SlowLogTaskImpl implements SlowLogTask {
+  private final String name;
+  private long startMS;
+  private long endMS;
+
+  public SlowLogTaskImpl(String name) {
+    this.name = name;
+    this.startMS = 0;
+    this.endMS = 0;
+  }
+
+  @Override
+  public void start() {
+    this.startMS = System.currentTimeMillis();
+  }
+
+  @Override
+  public void end() {
+    this.endMS = System.currentTimeMillis();
+  }
+
+  @Override
+  public String getSlowLogString(boolean pretty) {
+    String sep = pretty ? "\n" : "";
+    StringBuilder sb = new StringBuilder();
+    sb.append("{").append(sep);
+    sb.append("\"name\":").append("\"").append(name).append("\"").append(sep);
+    sb.append(",\"start\":").append("\"").append(getStartString()).append("\"").append(sep);
+    sb.append(",\"end\":").append("\"").append(getEndString()).append("\"").append(sep);
+    sb.append(",\"duration\":").append("\"").append(getDurationString()).append("\"").append(sep);
+    sb.append("}");
+    return sb.toString();
+  }
+
+  private String getStartString() {
+    if (startMS == 0) {
+      return "NotStarted";
```

Comment:
```suggestion
      return "N/A";
```

N/A stands for not not applicable, maybe it's more clear and concise